frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/CheckOutBranchAction.java

```diff
@@ -54,20 +53,21 @@ public void actionPerformed(AnActionEvent anActionEvent) {
       return;
```

Comment:
So why don't we perform this check in the `update` making an action` impossible to perform in the first place?

Reply:
Done, added a (rather defensive) check in `update` as well